include/common/s1ap_structs.h

```diff
@@ -747,7 +749,7 @@ typedef enum security_integrity_algo {
 }security_integrity_algo;
 
 
-#define BUFFER_SIZE 255
+#define BUFFER_SIZE 2056
```

Comment:
I would say move all the code of monitoring to separate header file. and keep all function declaration on that file. 
Also function definitions also can be moved to separate file which handles only monitoring request coming from the prometheus.

Reply:
done